components/dex/values.yaml

```diff
@@ -110,3 +110,9 @@ envVars:
       secretKeyRef:
         name: grafana-sso
         key: client-secret
+
+ingress:
```

Comment:
Removed the separate ingress resource definition and add the one from the helm-values as that allows us to override the configuration in the deploy repo.

Reply:
I fixed up the existing appsets to work with this.